Source/ThirdParty/AngelScript/include/angelscript.h

```diff
@@ -40,8 +40,8 @@
 #ifndef ANGELSCRIPT_H
 #define ANGELSCRIPT_H
 
```

Comment:
This header is from third-party. If you modified it then following our existing convention I would recommend you put your name in the “Modified for Urho3D by” line at the top of the file.